### ISSUE TITLE
Docker build error

### ISSUE THREAD

**Issue by nleguillarme** [RMLio/rmlmapper-java#106]
Building the docker image using `docker build -t rmlmapper:latest .` throws the following error :

```
Step 6/8 : RUN mv `find target/ -iname rmlmapper*.jar;` /rmlmapper.jar
 ---> Running in 440a9ea427f8
mv: can't rename 'target/rmlmapper-4.9.2-all.jar': No such file or directory
mv: can't rename 'target/rmlmapper-4.9.2-r349.jar': No such file or directory
The command '/bin/sh -c mv `find target/ -iname rmlmapper*.jar;` /rmlmapper.jar' returned a non-zero code: 1
```

**Comment by pheyvaer** [comment]
Hi @nleguillarme 

Thanks for the issue! We are aware of this and are looking into a fix.

**Comment by DylanVanAssche** [comment]
We fixed this issue in v4.9.3! Thanks for using RML and our tools!
